monkey/infection_monkey/monkey.spec

```diff
@@ -67,17 +67,11 @@ def process_datas(orig_datas):
 
 
 def get_binaries():
-    binaries = get_windows_only_binaries() if is_windows() else get_linux_only_binaries()
+    binaries = [] if is_windows() else get_linux_only_binaries()
```

Comment:
Don't we need this?
What happens if I compile on Win10 and try to run it on Win7?

Reply:
IDK, I haven't tested. I'll try to test and let you know.

Reply:
I tried freezing on win10 and then ran on server 2008, seemed to work just fine.